prime-router/metadata/hl7_mapping/ORU_R01/base/observation-result.yml

```diff
@@ -0,0 +1,125 @@
+name: ORU-R01-observation-result
+constants:
+  hl7OBXField: /PATIENT_RESULT/ORDER_OBSERVATION(%{orderIndex})/OBSERVATION(%{resultIndex})/OBX
+elements:
+  - name: result-set-ID
+    value: [ '%resultIndex + 1' ]
+    hl7Spec: [ '%{hl7OBXField}-1' ]
+
+  - name: result-value-type-st
+    value: [ '''ST''' ]
+    hl7Spec: [ '%{hl7OBXField}-2' ]
+
+  - name: result-value-type-cwe
+    condition: '%resource.value.exists()'
```

Comment:
Change this to `%resource.value is CodeableConcept`

Reply:
fixed